From 8c335a207a6e0acd7a1813d5d4a1beb494e773a3 Mon Sep 17 00:00:00 2001
Message-ID: <8c335a207a6e0acd7a1813d5d4a1beb494e773a3.1733602367.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1733602367.git.sharlatanus@gmail.com>
References: <cover.1733602367.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 7 Dec 2024 00:51:15 +0000
Subject: [PATCH 3/4] gnu: Add rust-katex-doc-0.1.

* gnu/packages/crates-io.scm (rust-katex-doc-0.1): New variable.

Change-Id: I62b79193b8b7764bffeb0dba23e110a34a705416
---
 gnu/packages/crates-io.scm | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/gnu/packages/crates-io.scm b/gnu/packages/crates-io.scm
index cf5e80b44e..d2eeb5a71d 100644
--- a/gnu/packages/crates-io.scm
+++ b/gnu/packages/crates-io.scm
@@ -34258,6 +34258,24 @@ (define-public rust-jwalk-0.8
 speed then walkdir provides a smaller and more tested single threaded implementation.")
     (license license:expat)))
 
+(define-public rust-katex-doc-0.1
+  (package
+    (name "rust-katex-doc")
+    (version "0.1.0")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (crate-uri "katex-doc" version))
+       (file-name (string-append name "-" version ".tar.gz"))
+       (sha256
+        (base32 "1fa0fi3bv9qdab0cb49q8971vszf0nzmvnvm70lns5xrpyyq0nqy"))))
+    (build-system cargo-build-system)
+    (home-page "https://github.com/CAD97/katex-doc")
+    (synopsis "XaaS for KaTeX on docs.rs")
+    (description
+     "This package provides a XSS as a Service for KaTeX on docs.rs")
+    (license (list license:unlicense license:expat))))
+
 (define-public rust-koibumi-base32-0.0.2
   (package
     (name "rust-koibumi-base32")
-- 
2.46.0

